styles/style.css

```diff
@@ -0,0 +1,205 @@
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background: #eceef1;
+  font-family: arial, sans-serif;
+  color: #4c4c4c;
+}
+
+a {
+  text-decoration: none;
+  color: #0077c5;
+}
+
+/* HEADER SECTION */
```

Comment:
Please remove all comments from your CSS code. This is required by Microverse to approve the project, and it makes your code look clean! ✨